File: jm-redis-cacheable/src/main/java/com/example/rediscacheable/server/mapper/UserMapper.java
package com.example.rediscacheable.server.mapper;


import com.baomidou.mybatisplus.core.mapper.BaseMapper;
import com.example.rediscacheable.server.domain.User;
import org.apache.ibatis.annotations.Mapper;

@Mapper
public interface UserMapper extends BaseMapper<User> {

}

File: kg-redis-cacheable/src/main/kotlin/com/example/rediscacheable/server/mapper/UserMapper.kt
package com.example.rediscacheable.server.mapper

import com.baomidou.mybatisplus.core.mapper.BaseMapper
import com.example.rediscacheable.server.domain.User
import org.apache.ibatis.annotations.Mapper

/**
 * @author : songtc
 * @since : 2023/12/08 14:19
 */
@Mapper
interface UserMapper : BaseMapper<User> {
}